frontend/src/__mocks__/mockSecretK8sResource.ts

```diff
@@ -32,7 +32,7 @@ export const mockSecretK8sResource = ({
   data: {
     AWS_ACCESS_KEY_ID: 'c2RzZA==',
     AWS_DEFAULT_REGION: 'dXMtZWFzdC0x',
-    AWS_S3_BUCKET: '',
+    AWS_S3_BUCKET: 'test-bucket',
```

Comment:
If we wanna test wether some data connections has this value and others don't you can always parametrize this values and then call what you need.